components/brave_perf_predictor/browser/bandwidth_linreg.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_PREDICTOR_H_
+#define BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_PREDICTOR_H_
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "base/containers/flat_map.h"
+#include "brave/components/brave_perf_predictor/browser/bandwidth_linreg_parameters.h"
```

Comment:
move to .cc?

Reply:
this is where `feature_count` is defined, deliberately so to catch any potential changes to the number of features used in the model.